### ISSUE TITLE
Error reading lang file on start

### ISSUE THREAD

**Issue by WTell** [dubocr/homebridge-tahoma#389]
Homebridge throws the following at me when started:

```
[Tahoma] Fail to load lang file: TypeError: Cannot read properties of undefined (reading 'toLowerCase')
     at Platform.loadLocation (/usr/local/lib/node_modules/homebridge-tahoma/src/Platform.ts:83:54)
     at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

How do I progress debugging this? Some devices have special Norwegian characters (Æ, Ø, Å) in their names in the TaHoma Web portal.

**Comment by trolli-ch** [comment]
Also I have problems with the lang file. So the Plugin starts but doesn't show the devices in the Home app.

[9/8/2022, 9:42:28 AM] [TaHoma] Configure device Regensensor
[9/8/2022, 9:42:28 AM] [TaHoma]   RainSensor > RainSensor
[9/8/2022, 9:42:28 AM] [TaHoma] Configure device Sonnensensor
[9/8/2022, 9:42:28 AM] [TaHoma]   LightSensor > LuminanceSensor
[9/8/2022, 9:42:28 AM] [TaHoma] Configure device Markise Dach
[9/8/2022, 9:42:28 AM] [TaHoma]   Awning > PositionableHorizontalAwning
[9/8/2022, 9:42:28 AM] [TaHoma] Configure device Markise links
[9/8/2022, 9:42:28 AM] [TaHoma]   ExteriorScreen > PositionableScreen
[9/8/2022, 9:42:28 AM] [TaHoma] Configure device Fenster links
[9/8/2022, 9:42:28 AM] [TaHoma]   Awning > PositionableHorizontalAwningUno
[9/8/2022, 9:42:28 AM] [TaHoma] Configure device Markise vorne
[9/8/2022, 9:42:28 AM] [TaHoma]   ExteriorScreen > PositionableScreen
[9/8/2022, 9:42:28 AM] [TaHoma] Configure device Fenster rechts
[9/8/2022, 9:42:28 AM] [TaHoma]   Awning > PositionableHorizontalAwningUno
[9/8/2022, 9:42:28 AM] [TaHoma] Fail to load lang file: Error 400 Unknown object. (UNSPECIFIED_ERROR)

**Comment by trolli-ch** [comment]
[9/8/2022, 11:58:58 AM] [TaHoma] Fail to load lang file: TypeError: Cannot read properties of undefined (reading 'toLowerCase')
    at Platform.loadLocation (/volume1/homebridge/node_modules/homebridge-tahoma/src/Platform.ts:83:54)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)

**Comment by dubocr** [comment]
Hi,

This error should have no impact on service. It is fixed in release 2.2.36 but if no device appear there is another issue.

**Comment by WTell** [comment]
The issue was indeed fixed in 2.2.36! Thanks!
